src/style.css

```diff
@@ -1330,12 +1330,17 @@ a {
   .transform-inputs-container {
     margin-top: 50px;
   }
+
+  @supports (-webkit-appearance: none) {
+    .input-range-container {
+      flex-direction: column;
+      gap: 10px;
+    }
+  }
```

Comment:
why this and why isn't the webkit not added to the rule specifically and it should look like this:
![image](https://github.com/Dun-sin/Code-Magic/assets/78784850/4a885845-4e48-43c7-9d5b-8511811fa232)

Reply:
@Dun-sin Also for me on firefox browser I get the same issue

Reply:
Using the property on the original rule does the same thing, you can Google how it works as a property or find how it's done in this project by using ctrl + f and then webkit

Reply:
Also I dont know whats happening but the width of the checkmark changes when the screen width is less than 1000px

Reply:
So Should I just increase the width of the checkmark in the media query of max-width 1000px as it seems to fix the issue and get the desired result like the one you want.

Reply:
@Dun-sin Also there isn't a -webkit-width rule that I can directly apply. So I would need to use the support rule to target chromium based browsers